test/browser_test.dart

```diff
@@ -0,0 +1,51 @@
+@Tags(['browser'])
+
+import 'package:js/js.dart';
+import 'package:node_interop/js.dart';
+import 'package:node_interop/util.dart';
+import 'package:sass/src/node/compile_options.dart';
+import 'package:test/test.dart';
+import 'ensure_npm_package.dart';
+import 'package:sass/src/node/compile_result.dart';
+
+@JS()
+external Sass get sass;
+
+@JS()
+class Sass {
+  external NodeCompileResult compileString(String text,
+      [CompileStringOptions? options]);
+  external Promise compileStringAsync(String text,
+      [CompileStringOptions? options]);
+}
```

Comment:
@nex3 I'm using `@JS()` instead of `context['sass']`, but because of the imports I brought in to type it, I'm getting the following error:

```
lib/src/node/url.dart:11:7:
Error: Non-static JS interop class 'JSUrl' conflicts with natively supported class 'Url' in 'dart:html'.
class JSUrl {
      ^
Error: Compilation failed.
```

I tried going over the codebase and using `Url` from `dart:html`, but it's missing the constructor, which I assume is why we have our own class in the repo. Not sure where to go next 😅

Reply:
Do you need `dart:html` at all here? In theory the browser support should just be a subset of Node support rather than containing anything browser-specific. Or does `dart:html` get brought in by the test framework itself?

Reply:
I'm not including it directly in the test, but it seems to only happen during test runs. I think we're hitting this [bug](https://github.com/dart-lang/sdk/issues/42200), specifically:v

> The users noticed that the API is hidden, so they tried using @JS to access it directly. This creates an issue because dart2js associates the dart:html interceptor with those types, so their JS-interop interfaces don't work

Reply:
Actually, it's possible you can get away with using the `dart:html` bindings even in Node if you just use `import 'dart:html' show Url`. That's probably worth at least trying before you start working around this.

Reply:
Hm, I added `@anonymous` in https://github.com/oddbird/dart-sass/commit/7e51c9b326a1f5e6df71cd4e959bc85359ee84ed, but didn't mess with `jsEval()` and the tests seem to be passing. @nex3 Does that make sense?

Reply:
If it works, it works! JS interop is an eternal mystery.